code/modules/trader/trade_system.dm

```diff
@@ -18,6 +18,10 @@ var/datum/subsystem/trade_system/SStrade
 	NEW_SS_GLOBAL(SStrade)
 
 /datum/subsystem/trade_system/Initialize(timeofday)
+	for(var/obj/structure/trade_window/TW in processing_objects)
```

Comment:
what

Reply:
This is related to the problem we had before; trade_window is mapped in so it's already in processing_objects but I want it to be added to this list inside the subsystem. There are three ways to do this:
1. Inside trade window, have it check every tick if the subsystem exists and if it's not part of its list, add itself
2. When the subsystem is initialized, look for trade windows that were mapped and add them (this)
3. Have the list all_twindows be global instead of a list on the subsystem
I prefer option 2

Reply:
Why not do the things you do inside the trade window's New in its `initialize` instead?